arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,1127 @@
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Workspace collection updated successfully",
+                        "schema": {
+                            "$ref": "#/definitions/WorkspaceCollection"
+                        }
+                    },
+                    "400": {
+                        "description": "Request is not well formed"
+                    },
+                    "403": {
+                        "description": "Authentication failed"
+                    },
+                    "404": {
+                        "description": "Subscription, resource group, or workspace collection does not exist"
+                    },
+                    "408": {
+                        "description": "Operation could not be completed within the permitted time"
+                    },
+                    "409": {
+                        "description": "Conflicting change"
+                    },
+                    "503": {
+                        "description": "Service unavailable"
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                    "WorkspaceCollections"
+                ],
+                "operationId": "deleteWorkspaceCollection",
+                "description": "The **Delete Power BI Workspace Collection** operation deletes a Power BI Workspace Collection.",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "subscriptionId",
+                        "in": "path",
+                        "description": "Azure subscription id",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "resourceGroupName",
+                        "in": "path",
+                        "description": "Azure resource group",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "workspaceCollectionName",
+                        "in": "path",
+                        "description": "Power BI Embedded workspace collection name",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "api-version",
+                        "in": "query",
+                        "description": "Azure API version",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "202": {
```

Comment:
usually an operation with 202 response in long running. Just verifying if this and other operations where 201, 202 are specified, are they long running?